sdk/eventhub/Azure.Messaging.EventHubs/src/Errors/EventHubsClientClosedException.cs

```diff
@@ -19,8 +19,8 @@ public sealed class EventHubsClientClosedException : EventHubsException
         /// <param name="resourceName">The name of the Event Hubs resource, such as an Event Hub, consumer group, or partition, to which the exception is associated.</param>
         /// <param name="message">The error message that explains the reason for the exception.</param>
         ///
-        internal EventHubsClientClosedException(string resourceName,
-                                                string message) : this(resourceName, message, null)
+        public EventHubsClientClosedException(string resourceName,
+                                              string message) : this(resourceName, message, null)
```

Comment:
Not unless we wanted to have to shift them to the Shared project, which seems excessive just to hide the constructor.    As a counter-point, any reason that you can see why it would be harmful to allow developers to construct these?

Reply:
Not harmful. Just thought we should avoid it if we could, but you have a point.